force-app/main/default/lwc/similarProducts/__tests__/similarProducts.test.js

```diff
@@ -0,0 +1,19 @@
+import { createElement } from 'lwc';
```

Comment:
This test fails (as it's just the scaffolded template). Should it be part of another PR?

Reply:
I will try and remove it again - for some reason when I checkout from `prerelease/summer20` it won't remove this file.